docs/user_guide/algorithm_list.rst

```diff
@@ -139,6 +163,49 @@ Graphs are often too large to handle, so a portion of the graph is extracted. Of
     This algorithm finds a maximal subgraph that contains nodes of at least degree *k*.
 
 
+.. py:function:: subgraph.k_truss(graph: Graph(is_directed=False), k: int) -> Graph
+
+    Finds the subgraph of edges with are part of k-2 triangles.
+
+
+.. py:function:: subgraph.maximal_independent_set(graph: Graph) -> NodeSet
+
+    Finds a maximal set of independent nodes, meaning the nodes in the set share no edges with each other
+    and no additional nodes in the graph can be added which satisfy this criteria.
+
+
+.. py:function:: subgraph.subisomorphic(graph: Graph, subgraph: Graph) -> bool
+
+    Indicates whether subgraph is an isomorphic subcomponent of graph.
```

Comment:
In other parts of the documentation (e.g. in [`k-core`](https://metagraph.readthedocs.io/en/latest/user_guide/algorithm_list.html#subgraph.k_core)), parameters mentioned in the description are italicized. I think we should do the same here for `graph` and `subgraph`.

Reply:
Reviewing the current docs has shown that we're pretty inconsistent in how we visually different parameter names. 
[`k-core`](https://metagraph.readthedocs.io/en/latest/user_guide/algorithm_list.html#subgraph.k_core) italicizes the parameter names. 

[`betweeness_centrality`](https://metagraph.readthedocs.io/en/latest/user_guide/algorithm_list.html#centrality.betweenness) shows the parameter name in a different color. 

Either choice seems sufficient as long as the documentation is consistent about it.

Reply:
I went with double backticks to color the parameter names rather than use italics or bold. They should all be consistent now.